vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -13902,7 +13906,11 @@ unit:KiloBYTE
   qudt:applicableSystem sou:IMPERIAL ;
   qudt:applicableSystem sou:PLANCK ;
   qudt:applicableSystem sou:SI ;
-  qudt:conversionMultiplier 5545.17744447956 ;
+  # WRONG MULTIPLIER  : unit:KiloBYTE - calculated from factors: 5678.261703147071974745965538985370, actual: 5545.17744447956
+  #
+  #  ──┬ unit:KiloBYTE multiplier: 5545.17744447956 (correct: 5678.26170314707197474596553898537)
```

Comment:
If BYTE is 5.545... why isn't KiloBYTE 5545...?

Reply:
Are you conflating KiloBYTE with KibiBYTE?

Reply:
Thanks for spotting this. No clue how that happened.

Reply:
It was caused by `unit:KiloBYTE` having the `prefix:Kibi` instead of `prefix:Kilo`. Fixed in subsequent commit.

Reply:
Do you depend on qudt:prefix? I notice that it is not consistently present.

Reply:
Also, the use of qudt:prefix would be ambiguous with compound units such as unit:KiloGM-PER-MegaBTU_IT.